internal/util/valueparser.go

```diff
@@ -52,6 +52,25 @@ func (vp *ValueParser) PInt64() *int64 {
 	return &v
 }
 
+// PUInt64 interprets the underlying value as an uint64 and returns a pointer to
+// that value.
+func (vp *ValueParser) PUInt64() *uint64 {
```

Comment:
Mind adding a test for this new method as well? You can probably just duplicate the PInt64 cases.

Reply:
Done.